src/test/kotlin/com/gamingbacklog/api/gamingbacklogapi/unit/services/UserServiceTests.kt

```diff
@@ -0,0 +1,157 @@
+package com.gamingbacklog.api.gamingbacklogapi.unit.services
```

Comment:
Note: these tests aren't complete. The remainder of these tests will be added in https://gaming-backlog.atlassian.net/browse/GB-79